lib/chef_backup/helpers.rb

```diff
@@ -69,6 +69,15 @@ def log(message, level = :info)
     ChefBackup::Logger.logger.log(message, level)
   end
 
+  # For some reason, the restore codepath puts the shell_timeout flag at top
+  # level but the backup codepath puts it underneath the service_config
+  # hierarchy. Pick the one we find.
```

Comment:
We chatted about this and decided the long tail of changes probably isn't worth it right now.